src-plugins/itkFilters/itkFiltersErodeProcess.h

```diff
@@ -13,24 +13,17 @@
 
 #pragma once
 
-#include <medAbstractData.h>
 #include <itkMorphologicalFiltersProcessBase.h>
 
-#include <itkFiltersPluginExport.h>
```

Comment:
Why do we remove that ?
Are you sure we can ?
I think at least Windows systems need it.

Reply:
This file includes itkMorphologicalFiltersProcessBase.h which includes itkFiltersProcessBase.h which includes itkFiltersPluginExport.h, so if your windows compiler has a problem with it I would change compilers (or better, OS) :P